base_agent/documents/json_schema/filters.schema.json

```diff
@@ -15,9 +15,7 @@
                 "ordinal": {
                     "anyOf": [
                         {
-                            "enum": [
-                                "FIRST"
-                            ]
+                            "$ref": "action_dict_components.schema.json#/definitions/fixed_value"
```

Comment:
assuming changes here are in sync with : https://github.com/facebookresearch/droidlet/issues/191

Reply:
Yep!